client/lib/signup/test/dependency-store.js

```diff
@@ -23,7 +23,7 @@ describe( 'dependency-store', function() {
 		SignupDependencyStore = require( '../dependency-store' );
 		SignupActions = require( '../actions' );
 
-		const store = createStore( reducer );
+		const store = createStore( combineReducers( initialReducers ) );
```

Comment:
why are we changing these thins here instead of where `initialReducers` is defined? it seems like a leak in design to force the dependents of the code to update to reflect what appears as internal changes…